storage/src/add_bucket_iam_member.php

```diff
@@ -29,15 +29,16 @@
 /**
  * Adds a new member / role IAM pair to a given Cloud Storage bucket.
  *
- * @param string $bucketName the name of your Cloud Storage bucket.
- * @param string $role the role you want to add a given member to.
- * @param string[] $members the member(s) you want to give the new role for the Cloud
- * Storage bucket.
- *
- * @return void
+ * @param string $bucketName The name of your Cloud Storage bucket.
+ * @param string $role The role to which the given member should be added.
+ * @param string[] $members The member(s) to be added to the role.
  */
 function add_bucket_iam_member($bucketName, $role, $members)
```

Comment:
For the same reason as above
```suggestion
function add_bucket_iam_member($bucketName, $role, array $members)
```